### PR TITLE
Sort localizable strings for English localization

### PR DESCRIPTION
**Pull request by shubhankar-mern**
This fixes the localicalization strings sorting order

### CODE REVIEW

**Comment by shubhankar-mern** [comment]
@kbhawkey @shannonxtreme plzz review and merge this

**Comment by sftim** [comment]
/retitle Sort localizable strings for English localization

File: data/i18n/en/en.toml [hunk 1]
```diff
@@ -160,6 +160,78 @@ other = "Tell your story"
...
+other = "Represents a common type of Kubernetes user."
+
+[layout_docs_glossary_workload_name]
+other = "Workload"
+
+[layout_docs_glossary_workload_description]
+other = "Applications running on Kubernetes."
+
 [layouts_docs_glossary_description]
 other = "This glossary is intended to be a comprehensive, standardized list of Kubernetes terminology. It includes technical terms that are specific to Kubernetes, as well as more general terms that provide useful context."
```
Comment (sftim) [inline data/i18n/en/en.toml:236]: `layouts_docs_glossary_description` collates ahead of `layout_docs_glossary_workload_description` (d comes before w in the alphabet).

**Review by sftim — CHANGES_REQUESTED** [review]
These need to end up fully sorted. Can you revise this to sort all the keys alpabetically @shubhankar-mern ?

File: data/i18n/en/en.toml (hunk 1 above)
Comment (shubhankar-mern) [reply data/i18n/en/en.toml:236]: [layout_docs_glossary_extension_name]
[layout_docs_glossary_extension_description]
here d occurs before n..so it should be before it right?

**Comment by shubhankar-mern** [comment]
@kbhawkey @sftim @shannonxtreme review and merge please..i have done the recommendee changes

File: data/i18n/en/en.toml [hunk 2]
```diff
@@ -160,24 +160,96 @@ other = "Tell your story"
...
 other = "Click on the"
 
+[layout_docs_glossary_community_name]
+other = "Community"
+
+[layout_docs_glossary_community_description]
+other = "Related to Kubernetes open-source development."
+
+[layout_docs_glossary_core-object_name]
+other = "Core Object"
```
Comment (kbhawkey) [inline data/i18n/en/en.toml:182]: Hello @shubhankar-mern .
How are you ordering the fields?
For example, should `layout_docs_glossary_core-object_name` appear before `layout_docs_glossary_core-object_description`

Comment (shubhankar-mern) [reply data/i18n/en/en.toml:182]: Object description comes before object name .....d before n @kbhawkey

**Comment by kbhawkey** [comment]
Duplicate of #39013.
/close

**Pull request closed without merging**